### PR TITLE
Badge and button documentation

### PR DESCRIPTION
**Pull request by kart-c**
## Badge and button documentation

- Added more button variants (icon button, floating action button, link)

### CODE REVIEW

**Review by Abhijeet43 — APPROVED** [review]
Code is Fine!!!!... Good to merge

File: Components/Badge/badge.html [hunk 1]
```diff
@@ -5,32 +5,101 @@
...
+					<script src="https://emgithub.com/embed.js?target=https%3A%2F%2Fgithub.com%2Fkart-c%2FCL-code-snippets%2Fblob%2Fmain%2Fbadge2.html&style=vs2015&showBorder=on&showCopy=on&fetchFromJsDelivr=on"></script>
+				</div>
+
+				<div class="main-footer">
+					<a href="../Avatar/avatar.html" class="previous-pg"
+						><i class="fas fa-arrow-left"></i> Avatar</a
+					>
+					<a href="../Button/button.html" class="next-pg"
+						>Button <i class="fas fa-arrow-right"></i
+					></a>
```
Comment (VishalPatil18) [inline Components/Badge/badge.html:100]: The code indentation looks broken here. Maybe you could fix this.

File: Components/Button/button-component.css [hunk 2]
```diff
@@ -49,3 +91,26 @@
...
+	height: 5.6rem;
+	width: 5.6rem;
+}
+
+.float-icon > i {
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translateX(-50%) translateY(-50%);
+}
```
Comment (VishalPatil18) [inline Components/Button/button-component.css]: An alternative which can be used;

```CSS
display: flex;
justify-content: center;
align-items: center;
```

or using Grid;
```CSS
display: grid;
place-items: center;
```

**Review by sagar455 — APPROVED** [review]
Good to go

**Review by VishalPatil18 — APPROVED** [review]
- A couple of code indentation fixes and alternative's added
- The `Button` button should be highlighted I guess on button page. Instead `Badge` is getting highlighted. You might want to correct that.

Screenshot 2022-02-04 at 18 33 34

> Apart from a few suggestions, the landing page is quite cool, and also the buttons and badge work great. Great stuff @kart-c 👍

File: Components/Badge/badge.html (hunk 1 above)
Comment (kart-c) [reply Components/Badge/badge.html:100]: I think I missed the button page I will fix it right away

File: Components/Button/button-component.css (hunk 2 above)
Comment (kart-c) [reply Components/Button/button-component.css]: Flex is much simpler, don't know how I forgot to use it

**Pull request merged**